lib/rex/proto/dcerpc/client.rb

```diff
@@ -296,6 +296,8 @@ def call(function, data, do_recv = true)
 
     raw_response = ''
 
+    sleep 3
```

Comment:
This should probably be handled in a better way...
It seems like a wait needs to occur for the DCE-RPC call.
If a request is sent early, the Windows host will respond with an error code that makes it seem like everything is broken.